ukis_pysat/raster.py

```diff
@@ -22,41 +24,46 @@
     )
     raise ImportError(str(e) + "\n\n" + msg)
 
-from ukis_pysat.members import Platform
-
 
 class Image:
-    def __init__(self, path=None, dataset=None, arr=None, dimorder="first"):
+
+    da_arr = None
+
+    def __init__(self, data, dimorder="first", crs=None, transform=None):

```

Comment:
@MWieland Fabian changed quite a lot here and it's much better IMO, please have a look. We're also breaking some compatibility here.